external/test-runtime/XUnit.Runtime.depproj

```diff
@@ -9,56 +9,65 @@
     <NugetRuntimeIdentifier Condition="'$(ArchGroup)' == 'armel'">win10-arm</NugetRuntimeIdentifier>
     <RidSpecificAssets>true</RidSpecificAssets>
     <OutputPath>$(RuntimePath)</OutputPath>
-    <XUnitRunnerPackageId Condition="'$(TargetGroup)' != 'netfx'">xunit.console.netcore</XUnitRunnerPackageId>
-    <XUnitRunnerPackageId Condition="'$(TargetGroup)' == 'netfx'">xunit.runner.console</XUnitRunnerPackageId>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == 'uapaot'">.NETStandard,Version=v2.0</NuGetTargetMoniker>
-    <NuGetTargetMonikerShort Condition="'$(TargetGroup)' == 'uapaot'">netstandard2.0</NuGetTargetMonikerShort>
-    <NuGetTargetMoniker Condition="'$(TargetGroup)' == 'uap'">.NETStandard,Version=v2.0</NuGetTargetMoniker>
-    <NuGetTargetMonikerShort Condition="'$(TargetGroup)' == 'uap'">netstandard2.0</NuGetTargetMonikerShort>
+    <XUnitRunnerPackageId>xunit.runner.console</XUnitRunnerPackageId>
+    <XUnitRunner>xunit.console</XUnitRunner>
+    <XUnitTestAdapterPackageId>xunit.runner.visualstudio</XUnitTestAdapterPackageId>
+    <XUnitAdapter Condition="'$(TargetGroup)' == 'netcoreapp'">xunit.runner.visualstudio.dotnetcore.testadapter</XUnitAdapter>
+    <XUnitAdapter Condition="'$(TargetGroup)' == 'uap' OR '$(TargetGroup)' == 'uapaot'">xunit.runner.visualstudio.uwp.testadapter</XUnitAdapter>
+    <XUnitAdapter Condition="'$(TargetGroup)' == 'netfx'">xunit.runner.visualstudio.testadapter</XUnitAdapter>
+    <TestPlatformHostPackageId>microsoft.testplatform.testhost</TestPlatformHostPackageId>
+    <TestPlatformHost>testhost</TestPlatformHost>
+    <NuGetTargetMoniker Condition="'$(TargetGroup)' == 'uap' OR '$(TargetGroup)' == 'uapaot'">.NETStandard,Version=v2.0</NuGetTargetMoniker>
+    <NuGetTargetMonikerShort Condition="'$(TargetGroup)' == 'uap' OR '$(TargetGroup)' == 'uapaot'">netstandard2.0</NuGetTargetMonikerShort>
     <!-- Coverlet related properties -->
     <CoverletPackageId>coverlet.msbuild</CoverletPackageId>
     <CoverletPackageVersion>2.2.1</CoverletPackageVersion>
+    <!-- In case the project language is not set -->
+    <Language Condition="'$(Language)' == ''">unknown</Language>
     <!-- Don't warn if some dependencies were rolled forward -->
     <NoWarn>$(NoWarn);NU1603</NoWarn>
   </PropertyGroup>
+  
   <ItemGroup>
-    <PackageReference Include="xunit.console.netcore">
-      <Version>$(AppXRunnerVersion)</Version>
+    <!-- xunit packages -->
+    <PackageReference Include="xunit">
+      <Version>$(XUnitPackageVersion)</Version>
     </PackageReference>
-    <PackageReference Include="Microsoft.DotNet.BuildTools.TestSuite">
-      <Version>$(XunitNetcoreExtensionsVersion)</Version>
+    <PackageReference Include="Microsoft.DotNet.XUnitExtensions">
```

Comment:
Will not converge these for now as I don't think it's worth introducing a separate property. Let's deal with this at a later point.

Reply:
I think it is worth because if at some point we rename this package again, we just change one property instead of 20 different sites.

Reply:
we use it in 3 places in corefx and once in buildtools. I don't want to touch it now as we are moving the test framework to arcade and this will change. also I don't want to change buildtools again.

Reply:
Fair enough. Something to keep in mind for next iteration though.